hudi-client/src/main/java/org/apache/hudi/index/HoodieIndex.java

```diff
@@ -77,15 +80,15 @@ protected HoodieIndex(HoodieWriteConfig config) {
    * present).
    */
   public abstract JavaRDD<HoodieRecord<T>> tagLocation(JavaRDD<HoodieRecord<T>> recordRDD, JavaSparkContext jsc,
-      HoodieTable<T> hoodieTable) throws HoodieIndexException;
+                                                       HoodieTable<T> hoodieTable) throws HoodieIndexException;
```

Comment:
why these changes?

Reply:
I am not sure. my previous patches didn't have any such refactoring changes. I just do the regular intellij refactoring.